packages/playwright-test/src/dispatcher.ts

```diff
@@ -495,7 +496,7 @@ let lastWorkerIndex = 0;
 class Worker extends EventEmitter {
   private process: child_process.ChildProcess;
   private _hash: string;
-  private parallelIndex: number;
+  public parallelIndex: number;
```

Comment:
can this also be readonly to be consistent with workerIndex?

Reply:
done